src/lib/secrets.d.ts

```diff
@@ -2,14 +2,33 @@ import { Context } from '../function/context'
 import { Handler } from '../function/handler'
 import * as services from './services.json'
 
+export type Service = {
+  friendlyServiceName: string
+  service: string
+  isLoggedIn: boolean
+  bearerToken: string | null
+  grantedScopes: Array<{
+    scope: string
+    scopeInfo: {
+      category: string | null
+      scope: string
+      display: string
+      isDefault: boolean
+      isRequired: boolean
+      description: string | null
+      title: string | null
+    }
+  }> | null
+}
+
 export type Services = typeof services
 
 export type ServiceKey = keyof Services
 
-export type ServiceTokens<T extends ServiceKey> = Services[T]['tokens']
+export type ServiceTokens = Service
```

Comment:
I don't think we need to maintain backward-compatibility in a prerelease. Go ahead an remove this.